src/components/Score.js

```diff
@@ -3,16 +3,40 @@ import { browserHistory } from 'react-router';
 
 import ScoreInput from './ScoreInput.js';
 
+const pushPayload = {
+  sessionId: null,
+  scopes: {}
+};
+
 const Score = React.createClass({
+  updatePushPayload(scope, score, comment) {
+    pushPayload.sessionId = this.props.params.sessionId;
```

Comment:
The reason I put it in Score is b/c it's responsible for POSTing three ScoreInputs worth of data. Some of your other comments have me thinking about naming/wrapping this differently though, so let's put a pin in this while I refactor.